code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm

```diff
@@ -81,8 +85,9 @@
 		uplink.telecrystals += tc_per_nukie
 		tc_to_distribute -= tc_per_nukie
 
+
 	for (var/mob/living/L in orphans)
-		var/TC = new /obj/item/stack/telecrystal(user.drop_location(), tc_per_nukie)
+		var/TC = new /obj/item/stack/telecrystal(L.drop_location(), tc_per_nukie)
```

Comment:
I think adding both fixes to the changelog or the PR text would be great!

Reply:
Sure, only changed it since I didnt have a ref to user